### DISCUSSION TITLE
Add Moonbeam and Moonriver to supported chains

### DISCUSSION THREAD

**Discussion by eshaben** [wevm/wagmi#781] [Ideas]
Currently you can configure [Moonbeam](https://moonbeam.network/networks/moonbeam/) and [Moonriver](https://moonbeam.network/networks/moonriver/) with the `jsonRpcProvider`, but it would be nice to add these chains as a part of the constant chains

**Comment by tmm** [comment]
Thanks for the feedback! We aren't adding new chains to the core right now until we figure out a scalable process. In the meantime, you can create your own chains by conforming to the [`Chain`](https://github.com/wagmi-dev/wagmi/blob/main/packages/core/src/types/index.ts#L13) type.
